v22.2/striim.md

```diff
@@ -0,0 +1,111 @@
+
+{{site.data.alerts.callout_info}}
+To minimize downtime for your migration, configure a separate [continuous replication](#continuous-replication) application before you deploy the initial load application. Once the initial load is complete, deploy the continuous replication application.
+{{site.data.alerts.end}}
+
+Deploy this application to perform the initial load of data to CockroachDB. Remember that you should have already [created the schema objects](#before-you-begin) on CockroachDB.
+
+### Continuous replication
+
+To perform continuous replication of ongoing changes, create another Striim application and configure the source database using one of the **CDC** sources. Configure CockroachDB as a **PostgreSQL** target. For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster](connect-to-the-database.html). Do the following before deploying the application:
+
+- Configure the CDC reader as described in the [Striim documentation](https://www.striim.com/docs/en/switching-from-initial-load-to-continuous-replication.html).
+
+- Set up your source database for continuous replication as described in the [Striim for BigQuery documentation](https://www.striim.com/docs/GCP/StriimForBigQuery/en/connect_source-select.html).
+
+- Repeat the guidance for creating the [initial load](#initial-load) application:
+
+	- When configuring CockroachDB as a target, specify the **Connection URL** in [JDBC format](connect-to-the-database.html?filters=java&#step-5-connect-to-the-cluster) while appending the `reWriteBatchedInserts=true` property, and without specifying the username.
+
+	- After creating the target, [export the application](https://www.striim.com/docs/platform/en/hands-on-quick-tour.html#UUID-a846e232-87e4-d88b-eb77-fa80691bbdf7) and add the field `_h_ConnectionRetryCode: '40001'` to the TQL file. Then [import the modified TQL file](https://www.striim.com/docs/platform/en/creating-apps-by-importing-tql.html) to create a new application.
+
+Deploy this application once the [initial load](#initial-load) application has finished running. 
+
+## Replicate data from CockroachDB to a secondary source
+
+You can use Striim to replicate ongoing changes from CockroachDB to a secondary source. This may include a [downstream sink](changefeed-sinks.html) such as Kafka or cloud storage for purposes such as reporting, caching, or full-text indexing. For a list of targets, see the [Striim documentation](https://www.striim.com/docs/en/targets.html).
+
+To perform continuous replication of ongoing changes, create a Striim application, configure CockroachDB as a **PostgreSQL CDC** source, and select an appropriate downstream target. For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster](connect-to-the-database.html).
```

Comment:
Should we have a more expanded set of steps here? Have we gone through and tested using this for something like kafka?